ACN-Coding-Test/laravel-coding-test-level-1
# First Test - CRUD REST API

### What does this pull request fix/change?
This PR is for implementing CRUD - REST API for Events' 

### Testing steps
- Pull down this branch
- Run `composer install` and `php artisan server`
- Go to https://www.postman.com/collections/87063f8da8bcc325ad4d and import it to Postman
- Check that each and every request is working correctly.
